Documentation/VMR-re-bootstrapping.md

```diff
@@ -0,0 +1,53 @@
+# How to re-bootstrap the toolset used to build the VMR
+
+.NET utilizes itself to build therefore in order to build .NET from source, you
+first need to acquire or build a bootstrapping .NET SDK and other tooling such
+as arcade. Re-bootstrapping is the term used to describe when the bootstrapped
+toolset need to be updated. This document describes the steps to re-bootstrap
+the VMR.
+
+# When is it appropriate to re-bootstrap?
+
+As part of the release process, the toolset is updated (e.g. PRs are created via
+the release automation). Outside of a release, re-bootstrapping is only permitted
+during preview releases. It is not allowed during RC, GA, or servicing releases.
+The reason it is not allowed during non-preview releases is because of the negative
+impact it has on Linux distro maintainers who source build .NET. It is often a long
+and time consuming process for them to re-bootstrap. It is likely to cause
+significant delays in the release/availability of .NET within the distros that are
+source built.
+
+# Why is re-bootstrap necessary?
+
+Re-bootstrapping is necessary when .NET takes a dependency on new functionality
+added within the bootstrap toolset. For example suppose a new compiler feature is
+added. In order for a repo to take a dependency on the new feature, a re-bootstrap
+would be necessary.
```

Comment:
Add something like: 
The implication of this, and the restrictions of when re-bootstrapping is allowed, means that repos should, in general, wait to take a dependency on a new toolset feature until after that feature has been released.